test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -643,3 +643,80 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
         $result.Error | Should BeNullOrEmpty
     }
 }
+
+Describe "Bug fixes" -Tags "Feature" {
+
+    Context "InFile parameter negative tests" {
+
+        $testCases = @(
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile ""'
+                ScriptBlock = {Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile ""}
```

Comment:
The expectation is that http://httpbin.org return a call. I 've open issue #3012 to track this.